__tests__/test-data.test.js

```diff
@@ -310,3 +314,32 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: deletes comment from database", () => {
+    return request(app)
+      .delete("/api/comments/1")
+      .expect(204)
+      .then(() => {
+        return db.query('SELECT * FROM comments').then(({rows}) => {
+          expect(rows.find(obj => obj.comment_id === 1)).toBe(undefined)
+        })
+      });
+  });
```

Comment:
I like this test, making a request to the database to ensure the comment has been deleted.

There would be a more simple way to go about it though - you can alter your query to only select the comment you're looking for, and then check the length of your response rather than needing to use a `.find`

You've also nested your `.then` blocks in this test - refactor it slightly to avoid this